docs/admin.rst

```diff
@@ -823,12 +800,20 @@ left off. However, if the same issue persists, you will need to investigate
 further.
 
 .. |Reset Passphrase| image:: images/manual/screenshots/journalist-edit_account_user.png
+   :alt: The account edition form allows change name, reset passphrase, and reset two-factor authentication.
```

Comment:
- edition form->editing form
- allows change name->allows admins to change name

Reply:
Done.